docs/modules/ROOT/partials/getting-started/con-registry-serdes-avro.adoc

```diff
@@ -4,43 +4,41 @@
 [id='registry-serdes-types-avro-{context}']
 = Configure Avro SerDe with {registry}
 
-{registry} provides Kafka client serializer and deserializer classes for Apache Avro to make using Avro as
-easy as possible:
-
-* `io.apicurio.registry.utils.serde.AvroKafkaSerializer`
-* `io.apicurio.registry.utils.serde.AvroKafkaDeserializer`
+[role="_abstract"]
+{registry} provides the following Kafka client serializer and deserializer classes for Apache Avro:
 
+* `io.apicurio.registry.serde.avro.AvroKafkaSerializer`
+* `io.apicurio.registry.serde.avro.AvroKafkaDeserializer`
 
 .Configure the Avro serializer
 
-You can configure the Avro serializer class in the following ways:
+You can configure the Avro serializer class with the following:
 
-* {registry} location as a URL
-* Artifact ID strategy 
+* {registry} URL
+* Artifact resolver strategy 
 * Global ID strategy 
```

Comment:
@smccarthy-ie sorry I think I did not mention this in the previous document, but there is no Global ID strategy anymore. That concept does not exist now, it's been replaced in part with the SchemaResolver. And the features that all our different GlobalIdStrategies had are now replaced with config properties of DefaultSchemaResolver

Reply:
@famartinrh Thanks, I removed the main section on global ID strategy and replaced with default schema resolver, but wasn't sure what to do with minor references like this. Removing them all now...